### PR TITLE
Page object model

### PR DESCRIPTION
**Pull request by Elizalzate**
Approach with POM and improving some of the selectors

### CODE REVIEW

File: cypress/integration/buy-shirt.spec.ts [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
+    adressStepPage.proceedToCheckout();
 
-    cy.get("#cart_navigation > button > span").contains("I confirm my order").click();
+    shippingStepPage.proceedToCheckout();
 
-    cy.get("#center_column > div > p > strong")
-        .should("have.text", "Your order on My Store is complete.");
+    paymentStepPage.payByBankWire();
+    paymentStepPage.confirmOrder();
+    paymentStepPage.getConfirmation();
```
Comment (holgiosalos) [inline cypress/integration/buy-shirt.spec.ts]: as this method is performing a validation, I suggest using a different name (option 1) or another approach if you want to preserve this name (option 2).

1. Change the name to validateConfirmationMessage and receive the message you want to verify as parameter. At the end the code will look like this: `paymentStepPage.verifyConfirmationMessage(......)` 
2. Modify the method to return the confirmation message and perform the assertion here. At the end the code will look like this: `paymentStepPage.getConfirmation().should(......)` 

Any option is ok :)

File: cypress/integration/buy-shirt.spec.ts [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
+const email = "aperdomobo@gmail.com";
+const pass = "WorkshopProtractor";
 
-    cy.get("#uniform-cgv").click();
-    cy.get("[name='processCarrier']").click();
+describe("Buy a t-shirt", () => {
+  it("then should be bought a t-shirt", () => {
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+    menuContentPage.addTShirt();
```
Comment (holgiosalos) [inline cypress/integration/buy-shirt.spec.ts]: the logic associated to this method should go on the `productListPage`

File: cypress/integration/buy-shirt.spec.ts [hunk 3]
```diff
@@ -1,25 +1,31 @@
...
+    menuContentPage.confirmSummary();
```
Comment (holgiosalos) [inline cypress/integration/buy-shirt.spec.ts]: the logic related to this method should go on `shoppingCartPage`

**Review by manuelq12 — APPROVED** [review]
LGTM

**Pull request merged**